sys-fs/cryptsetup/cryptsetup-1.7.3.ebuild

```diff
@@ -59,6 +59,10 @@ src_prepare() {
 	sed -i '/^LOOPDEV=/s:$: || exit 0:' tests/{compat,mode}-test || die
 	epatch_user && eautoreconf
 
+	if use libressl; then
+		epatch "${FILESDIR}/libressl.patch"
```

Comment:
don't do conditional patching (given how the C macro is designed)! And do the patching before `epatch_user`, preferably